package.json

```diff
@@ -436,6 +437,94 @@
             }
           }
         ]
+      },
+      {
+        "type": "edge-python",
+        "label": "IoT Edge (Python)",
+        "initialConfigurations": [
+          {
+            "name": "Remote Debug IoT Edge Module (Python)",
+            "type": "python",
+            "request": "attach",
+            "port": 5678,
+            "host": "localhost",
+            "logToFile": true,
+            "redirectOutput": true,
+            "pathMappings": [
+              {
+                "localRoot": "${workspaceFolder}/modules/<module-folder>",
+                "remoteRoot": "<Absolute path of the program in the docker container>"
+              }
+            ],
+            "windows": {
+              "pathMappings": [
+                {
+                  "localRoot": "${workspaceFolder}\\modules\\<module-folder>",
```

Comment:
The folder may not go to modules folder. Since in this scenario, we may expect user enter from normal python project. Please refer to the root file of other languages

Reply:
Updated